README.md

```diff
@@ -1,54 +1,54 @@
 
-
 ## Directory Structure
 
- * **.github**
-    *CI pipeline and PR/issue templates*
- * **cmake**
-    *CMake files to aid with building*
- * **documents**
-    *Research, design, and documentation*
- * **external**
-    *Where all our submodules are located*
- * **urc_analysis**
-    *Nodes used for scientific analysis*
- * **urc_gazebo**
-    *Helper nodes used for simulation purposes*
- * **urc_manipulation**
-    *Collection of nodes used for the robotic arm*
- * **urc_msgs**
-    *Custom ROS messages used in various packages*
- * **urc_platform**
-    *Manages our nanopb protocol buffers*
- * **urc_navigation**
-    *Collection of nodes that form our navigation stack*
- * **urc_perception**
-    *Collection of nodes that form our perception stack*
- * **urc_platform**
-    *Nodes that are platform specific and used to communicate with the hardware, ie. IMU, joystick and motor controller*
- * **urc_teleop**
-    *Rosbridge nodes and web interface for teleoperation*
- * **urc_util**
-    *A collection of utility nodes and classes*
-
+- **.github**
+  _CI pipeline and PR/issue templates_
+- **cmake**
+  _CMake files to aid with building_
+- **documents**
+  _Research, design, and documentation_
+- **external**
+  _Where all our submodules are located_
+- **urc_analysis**
+  _Nodes used for scientific analysis_
+- **urc_gazebo**
+  _Helper nodes used for simulation purposes_
+- **urc_manipulation**
+  _Collection of nodes used for the robotic arm_
+- **urc_msgs**
+  _Custom ROS messages used in various packages_
+- **urc_platform**
+  _Manages our nanopb protocol buffers_
+- **urc_navigation**
+  _Collection of nodes that form our navigation stack_
+- **urc_perception**
+  _Collection of nodes that form our perception stack_
+- **urc_platform**
+  _Nodes that are platform specific and used to communicate with the hardware, ie. IMU, joystick and motor controller_
+- **urc_dashboard**
+  _Rosbridge nodes and web interface for teleoperation_
+- **urc_util**
+  _A collection of utility nodes and classes_
 
 ## Installation Instructions
 
 You will need to be using Ubuntu 22.04 to run ROS2. This can be accomplished with any of the following methods:
```

Comment:
Make sure to add your setup documentation to this list so that it is easier to find.